msvc2017/FreeOrion/StdAfx.h

```diff
@@ -0,0 +1,104 @@
+#pragma once
+
+// I generously include all external headers I found in the project header files,
+// including std, boost and GG headers.
```

Comment:
omit the GG code, I think. It is not a stable third party library like boost or std.

Reply:
Thanks for your advice. I updated the PR to treat GiGi as an internal library. That means ommiting it from StdAfx.h, while using precompiled headers to compile it. I restrained from using precompiled headers for the GiGiSDL-Project, as it consists of only a single compilation unit.